.config-schema.json

```diff
@@ -302,6 +302,12 @@
                     "type": "float",
                     "default": "0.2",
                     "x-env-variable": "OPENFGA_TRACE_SAMPLE_RATIO"
+                },
+                "serviceName": {
+                    "description": "the name of the service in traces.",
```

Comment:
the service name included in sampled traces.

Reply:
done